### PR TITLE
Update Modify_an_image_to_a_single_layer.md

### PR DESCRIPTION
**Pull request by lalyos**
docker build can squash the image

### CODE REVIEW

**Comment by Govinda-Fichtner** [comment]
@lalyos How to you know this is about building new images with `docker build --squash`? 

When looking at this topic in the topic guide I came to the conclusion that this has to be about flattening an existing image, because it says "modify an image to a single layer". For an existing image this cannot be done with `docker build`, right?

**Comment by lalyos** [comment]
the previous questions were about how build an image, describe Dockerfile options, and so on. So my assumption was that we have a Dockerfile. All automated images have a Dockerfile.

But you are right the word "modify" seem to imply there is an already existing image. Still if there is an existing image, you can overwrite (retag) with `docker buid --squash`. Overwriting an existing images is modifying it.

**Comment by lalyos** [comment]
btw `docker export` is working with a container not with an image, you probably meant [docker save](https://docs.docker.com/engine/reference/commandline/save/).

**Comment by Govinda-Fichtner** [comment]
Well, maybe we need to clarify the meaning of "modify an image to a single layer" with the authors of the study guide.

In the meantime we could just add both with a note about how it is not clear what is meant.

Also while `docker export` works with a container, `docker image import` creates an image.
What do you think?

**Comment by lalyos** [comment]
I've updated the PR by having two options.

File: domain2/Modify_an_image_to_a_single_layer.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
+- create a container from the image: [docker create](https://docs.docker.com/engine/reference/commandline/create/)
+- export the container to a flattened tar: [docker container export](https://docs.docker.com/engine/reference/commandline/export/)  
+- load the image back: [docker image import](https://docs.docker.com/engine/reference/commandline/image_import/)  
 
-## Third Party Resources
 [Flatten a Docker container or image](https://tuhrig.de/flatten-a-docker-container-or-image/)
+
+## With rebuilding
+
+See option `--squash` at [docker buil](https://docs.docker.com/engine/reference/commandline/build/#options)
```
Comment (Govinda-Fichtner) [inline domain2/Modify_an_image_to_a_single_layer.md]: it should be "of [docker build] ..."

**Review by Govinda-Fichtner — APPROVED** [review]

**Pull request merged**